runtime/onert/api/python/package/infer/session.py

```diff
@@ -6,14 +6,35 @@ class session(BaseSession):
     """
     Class for inference using nnfw_session.
     """
-    def __init__(self, nnpackage_path, backends="cpu"):
+    def __init__(self, nnpackage_path: str = None, backends: str = "cpu"):
```

Comment:
We need to rename argument `nnpackage_path` to `path`. It is supporting not only nnpackage but model file (tflite, circle, tvn) now.

Reply:
Thanks for your information. I applied it.